apps/src/music/views/MusicView.jsx

```diff
@@ -673,6 +673,7 @@ class UnconnectedMusicView extends React.Component {
             this.props.isProjectLevel
           }
           analyticsReporter={this.analyticsReporter}
+          blocklyWorkspace={this.musicBlocklyWorkspace}
```

Comment:
I added this to MusicLabView so that we could access `.getCode()` to obtain the serialization. We technically don't need to do this if we use a direct call to `Blockly.serialization.workspaces.save` inside `showLevelBuilderSaveButton`.